src/main/java/org/jitsi/jicofo/ChatRoomRoleAndPresence.java

```diff
@@ -140,17 +140,15 @@ public void memberPresenceChanged(ChatRoomMemberPresenceChangeEvent evt)
             }
             conference.onMemberJoined(sourceMember);
         }
-        else if (ChatRoomMemberPresenceChangeEvent.MEMBER_LEFT.equals(eventType)
-            || ChatRoomMemberPresenceChangeEvent.MEMBER_KICKED.equals(eventType)
-            || ChatRoomMemberPresenceChangeEvent.MEMBER_QUIT.equals(eventType))
+        else if (evt instanceof Left || evt instanceof Kicked)
         {
             if (owner == sourceMember)
             {
                 logger.info("Owner has left the room !");
                 owner = null;
                 electNewOwner();
             }
-            if (ChatRoomMemberPresenceChangeEvent.MEMBER_KICKED.equals(eventType))
+            if (evt instanceof ChatRoomMemberPresenceChangeEvent.Kicked)
```

Comment:
Isn't this just `Kicked ` as the rest?

Reply:
Yep. I'll fix that in the next "clean up" PR.